src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Metadata/Settings/ContentSettings.cs

```diff
@@ -0,0 +1,6 @@
+namespace OrchardCore.ContentManagement.Metadata.Settings;
+
+public class ContentSettings
+{
+    public bool IsCodeManaged { get; set; }
```

Comment:
We talked about having something like a system part like this before. Don't we have it? I can't seem to find neither code nor issue though.

Reply:
We don't have such a thing. I think you are referring to #11737 and issue #11736

Reply:
That may be it indeed.

Please add a note about this to the release notes though, because this new option will go undetected to developers otherwise.

Reply:
@Piedone docs were added